### DISCUSSION TITLE
Web3 URL Demo Day

### DISCUSSION THREAD

**Discussion by luffythink** [IntensiveCoLearning/Web3-URL#189] [Announcements]
预在 2024 年 8 月 9 号举办第 1 期线上「Web3 URL Demo Day」分享活动，
具体时间信息请关注 [Web3 URL 残酷共学频道](https://t.me/LXDAO/8748)通知。

**Comment by luffythink** [comment]
❤️DemoDay 作品基本要求
1，基于残酷共学开源开放，项目需要有公开可用 GitHub 存储库的链接
2，有完整项目概述，建议包含在 GitHub 的 README 中
3，项目基于 Web3:// 协议，有一个可正常访问的 Web3 界面
4，项目可部署并在 Web3Q Galileo 上线，最终目标是实现全链应用

❤️评选基本流程：
a.每个小组派出一位代表， 线上进行 Demo 演示分享
b.评委根据作品工作量、完整度、技术实施、创新性、产品体验、用户价值等多维度进行打分
c.根据评委评分得出最终排名

**Comment by luffythink** [comment]
会议主题：Web3 URL 残酷共学 Demo Day 预告
会议时间：2024/08/09 21:00-22:00 (GMT+08:00) 中国标准时间 - 北京

点击链接入会，或添加至会议列表：
https://meeting.tencent.com/dm/9JvaHmWWVoBy

#腾讯会议：348-555-005

**Comment by zedz** [comment]
组队项目名：逆境无赖Outlaw Voyage
组队成员信息：Zedz、Punkcan、Iyi、Antigone、Hython、Max
项目GitHub Repo 链接：[https://github.com/OutlawVoyage/OutlawVoyage](https://github.com/OutlawVoyage/OutlawVoyage)

**Comment by luffythink** [comment]
组队项目名：Web3Sounds 微波山之声
组队成员信息：Oscar、Derick、CHENFANGC
项目GitHub Repo 链接：https://github.com/luffythink/Web3Sounds

**Comment by linghuccc** [comment]
项目名：MAS Awareness
项目介绍：MAS Awareness 原本是我在 BSC 链发行的 NFT 项目，接触到 web3:// 之后，就考虑把这个 NFT 项目转化成一个完全链上的 NFT 项目。
成员信息：Ric Li C
项目GitHub Repo 链接：https://github.com/linghuccc/mas_awareness_web3url

**Comment by nishuzumi** [comment]
项目名：Web3Url Binding
项目介绍：一个快速将合约调用转换成实际url调用的小工具
成员信息：Box
项目GitHub Repo 链接：https://github.com/nishuzumi/web3url-binding-gen

**Comment by luffythink** [comment]
Demo 分享演示：https://youtu.be/xNud1QOWjeg 
值得记忆的时刻，WICL1st 虽告一段落，但 buidling 不止。大家继续加油～
- 第一名：800 U -- Web3URL Binding
- 第二名：600 U -- 逆境无赖 Outlaw Voyage
- 第三名：400 U -- MAS Awareness
- 第四名：200 U -- 微波山之声 Web3Sounds
